src/cli.mjs

```diff
@@ -42,15 +50,41 @@ export async function run(_argvInput, _container) {
     let subMessage = chalk.reset(
       "Use 'fauna <command> --help' for more information about a command.",
     );
+    let epilogue = "";
 
     if (argvInput.length > 0) {
-      subMessage = chalk.red(e.message);
+      // If the error isn't one of our known errors, wrap it in a generic error message.
+      if (isUnknownError(e)) {
+        subMessage = chalk.red(
+          `An unexpected error occurred...\n\n${e.message}`,
+        );
+        epilogue = `\n${BUG_REPORT_MESSAGE}`;
+
+        logger.debug(`unknown error thrown: ${e.name}`);
```

Comment:
Can we print the whole error, not just the name? That would include the stack trace and might aid in debugging on our end if a customer reports something.

Reply:
I'll do another debug message with the output of util.inspect. It will not do much, but it could be helpful for any fauna driver errors that bubble up.

Reply:
The stack is written via the logger output later on, btw.